deferred-leveldown.js

```diff
@@ -8,7 +8,6 @@ function DeferredLevelDOWN (db) {
   AbstractLevelDOWN.call(this, '')
   this._db = db
   this._operations = []
-  this._iterators = []
```

Comment:
Ah, damnit, this would affect `reachdown` 😄 (currently detects `deferred-leveldown` by presence of `_operations` and `_iterators`)

Reply:
In the interest of moving on, I'll restore `this._iterators = []`, with a comment saying it's temporary.